docs/src/_getting-started/meltano-at-a-glance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Meltano at a Glance
+description: If you want to know why you should use Meltano, here's a quick overview of the basics of all of it.
+layout: doc
+weight: 1
+---
+
+This document is intended to give you enough technical understanding of Meltano to becoming excited about it and wanting to use it! It's not going to teach you how to use it, we got Tutorials & How Tos for that. When you're ready to start your first Meltano project, we recommend you dive right into our Tutorial.
+
+### What users say
+
+_"For us it's a better day at work when we can use Meltano."_ - Nino Müller, Head of Technology at Substring
+
+
+_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as the data built tool (dbt)"._  - Martin Morset
+
+## Meltano at a glance
+Welcome to your Open Source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
```

Comment:
```suggestion
Welcome to your Open Source DataOps Infrastructure! With Meltano you can move your data with 10x the developer experience while also managing all of the data tools in your stack. With Meltano, you can collaboratively build and improve your ideal data platform like a software project; spinning up a service or tool (Singer connectors, Airflow, dbt, Great Expectations, Snowflake, etc) and easily configure, deploy, and manage it through a single control plane.
```

I'd love to see this more focused on the data replication piece while still highlighting the entire platform aspect of it.

Reply:
Sure I'm all up for it. I'm still looking for the place where I can find the latest Marketing plug description, and am randomly copying the ones I find look the most recent.

Reply:
But, one general note: Are you tied to the use of the word "Singer" inside this context? Because the fact is, this is supposed to sell people, and using "Singer" will always be a bit confusing (you know, they might and likely don't know what Singer is). I would simply use "data connectors", and put another point of "built on the de-facto protocol for data movement: Singer" somewhere.